collections/_infographics/energy/electricity-czechia.md

```diff
@@ -3,15 +3,30 @@ layout:        infographic
+
+## Methodology used and origin of data
+
+* To calculate emissions, we used data on gross electricity production by individual types of power plants or fuels used from the annual report on the operation of the Czech electricity system in 2018<!--[ERO: Annual Report on the Operation of the EC of the Czech Republic 2018](http://www.eru.cz/documents/10540/4580207/Rocni_zprava_provoz_ES_2018.pdf/)-->. Based on these data, we calculated the corresponding emissions using the relevant IPCC coefficients ([IPCC 2014: ANNEX III, Technology-specific Cost and Performance Parameters](https://www.ipcc.ch/site/assets/uploads/2018/02/ipcc_wg3_ar5_annex-iii.pdf)). for its uniformity and data availability from different sources. For example, the authors of the [ElectricityMap](https://www.electricitymap.org/map?lang=en) project used a similar approach.
+* We have merged some of the data for simplicity and clarity. For example, the entry "Biomass power plants" includes production from biomass-fired power plants and electricity production from biogas. The item "Hydroelectric power plants" includes both production from hydroelectric and pumped storage power plants, etc.
+* The selected emission factors represent the mean value of the _"lifecycle emissions"_, including the emissions associated with the related infrastructure required to operate the respective sources. An example of the lifecycle emissions shown in the overview is the non-zero emissions for nuclear power plants.
+* The item "Losses" includes technological self-consumption of energy sources, losses in the network of the transmission system operator, losses in the networks of individual distribution system operators, consumption of pumped storage power plants and the balance difference.
+* The individual consumption categories are divided into the relevant sectors of the national economy according to their main <!--[-->CZ-NACE<!--](https://nacev2.com/en)--> code. <!--Details of this breakdown can be found in the relevant ERO Interpretative Opinion ([ERO Interpretative Opinion No 8/2018](http://www.eru.cz/documents/10540/3687211/Vykladove+stanovisko_ERU_8_2018.pdf/91a9c5ec-a0bd-48bf-a9a1-9da63c6a7ec9)). The item "Other" then includes local consumption of electricity producers (supply to own company or installation). -->
```

Comment:
Not sure what to do this item. [CZ-NACE - http://www.nace.cz](http://www.nace.cz) has an English section but it does not translate the index.

I started looking at possibly using https://nacev2.com/en as the Nomenclature of Economic Activities are common codes. However, I do not have a good answer how to deal with the second link: http://www.eru.cz/documents/10540/3687211/Vykladove+stanovisko_ERU_8_2018.pdf/91a9c5ec-a0bd-48bf-a9a1-9da63c6a7ec9 which is what this section is referring to when explaining the "Other"-item in the infographics.